README.Rmd

```diff
@@ -101,6 +101,16 @@ rOpenSci's Software Peer Review process is run by:
 * [Julia Gustavsen](https://github.com/jooolia), Agroscope
 * [Emily Riederer](https://github.com/emilyriederer), Capital One 
 
+Associate editors for statistical software are:
+
+- [Ben Bolker](https://github.com/bbolker), McMaster University, Canada
```

Comment:
Do you think we should add countries to the other editors as well?

Reply:
Oh yeah, sorry, i forgot to mention that. I suspect so, because I find the not listing of countries sometimes a bit global-north-presumptuous, and see no reason not to add them. It's either all or none, and things are less presumptive and more equal when all have them, I'd say.